addons/pos_hr/static/tests/tours/PosHrTourMethods.js

```diff
@@ -50,6 +50,21 @@ odoo.define('pos_hr.tour.PosHrTourMethods', function (require) {
                 },
             ];
         }
+        cashInOutVisible(visible=true) {
+            return [
+                {
+                    trigger: ".pos-branding",
+                    run: () => {
+                        const shown = $(".pos-branding .cash-move-button").length;
+                        if(shown != visible) {
+                            const shouldBeVisible = visible ? "" : " not";
+                            const isVisible = shown ? "" : " not";
+                            throw new Error("Cash In/Out button is" + isVisible + " shown while it should" + shouldBeVisible);
+                        }
+                    },
+                },
+            ];
+        }
```

Comment:
```suggestion
        cashInOutVisible(visible = true) {
            const trigger = visible
                ? ".pos-branding .cash-move-button"
                : ".pos-branding:not(:has(.cash-move-button))";
            return [{ trigger, run: () => {} }];
        }
```

Don't use 'run' to check presence or absence an element. Use the complete selector for the `trigger` and put empty `run` for "check".

Using `run` to check for element normally results to random runbot errors.